### ISSUE TITLE
Broken links on the Front Page

### ISSUE THREAD

**Issue by HumairAK** [operate-first/operate-first.github.io-old#337]
@Gregory-Pereira  pointed out that both of these links don't work and need to be fixed.

![image](https://user-images.githubusercontent.com/10904967/144909263-92bf5729-6247-4908-bdbe-26b55fc130ff.png)

Though I don't think the links they pointed to before made much sense either, which pages should these point to? Are these supposed to point to learning paths as mentioned here https://github.com/operate-first/SRE/issues/424#issuecomment-966001779 ?? Do we even have content pages for these buttons that make sense? Do we need to write them?

**Comment by HumairAK** [comment]
@quaid

**Comment by quaid** [comment]
Yes, this is a kinda Sev1 issue that our prominent call-to-action buttons are 404.

Can we return the links to working with the destination we had before?

It's true the links (used to) go to sub-optimal pages until something better is built. Optimal would be directed learning paths, sub-optimal is that we have a look-around-and-DIY approach currently. That's what we have for now, and there is active work to write the content to go into the mapping here: https://github.com/operate-first/community/issues/103

**Comment by quaid** [comment]
OK, I created operate-first/operations#338 to create new landing pages for both those CTA links.

In the interest of fixing broken prominent links, feel free to force fix any mistakes I made in the PR to send it live. I'll edit this comment with the Netlify URL when the build is ready.

EDIT: Here is the build, my changes seem to have worked as intended: https://deploy-preview-338--relaxed-goldstine-ebb96e.netlify.app/
